steal.lic

```diff
@@ -148,9 +148,12 @@ class Steal
   end
 
   def build_target_data(item, container)
+    @data_builder_mode_toggle = true if @data_builder_mode
```

Comment:
I think its confusing to have this buried in a function - is there a reason it cant be in setup?